### PR TITLE
Fixes!

### PR DESCRIPTION
**Pull request by benbjohnson**
## Overview

This pull request fixes:

- [x] Data node reload from metastore.
- [x] Broker snapshotting/restore
- [x] HTTP redirection to the broker leader

### CODE REVIEW

File: raft/log.go [hunk 1]
```diff
@@ -24,10 +24,10 @@ import (
 
 const (
 	// DefaultHeartbeatInterval is the default time to wait between heartbeats.
-	DefaultHeartbeatInterval = 50 * time.Millisecond
+	DefaultHeartbeatInterval = 150 * time.Millisecond
```
Comment (otoolep) [inline raft/log.go]: Are these changes deliberate?

Comment (benbjohnson) [inline raft/log.go]: Yeah, a lower heartbeat is easier for testing at this point and it spews out heartbeat errors at a third of the rate which is nice. :) we can adjust later.

**Comment by otoolep** [comment]
Tested by me, +1. All functionality looks good.

File: raft/log.go (hunk 1 above)
Comment (pauldix) [inline raft/log.go]: yeah, this should be a config setting
